molgenis_documentation/user_documentation.md

```diff
@@ -270,26 +223,350 @@ Go to the Upload menu. You now should see something like this:
+However we meant to search for chromosome 7, so lets click the filter in the active filters box, and change the value to 7.
+We now have all the values for chromosome 7 in the table, however the results are divided over several pages of results, we'd like to view them all in one screen; click the "rows per page" dropdown below the table and select "100" this will show 100 results per page.
+The "FILTER" column shows the same value for every line, we are not interested in this column so let's hide it by clicking the checkbox in front of "#CHROM" in the attribute selection tree.
+
+Click any header in the table to sort the data based on that column, click again to sort in the other direction.
+Click one of the lines in the data table to zoom to the position of this variant in the genome browser.
+Click the symbol in front of the "SAMPLES" column header to show the columns belonging to the samples.
+
+Click the magnifing glass in front of the dataline to show a report for that line. The default report is just showing all attribute values in a structured way. However as stated above all kinds of reports can be added at runtime by an admin.
+
+###### Data manipulation
+
+Click the edit icon and change the chromosome from 7 to 8 and save.
+Adding a row works the same way, only without the prefilled fields.
+Now lets click the red garbage bin icon in front of a line to delete this line from the entity.
+
+##### Annotation mod
+
+The anotator mod of the data explorer is the user interface to use the MOLGENIS annotator framework, which can also be used as a standalone commandline jar.
+The annotator framework is a system to add data from other resources to your genomic entities. For example pathogenicity prediction, allele frequencies and phenotype information.
+
+![Dataexplorer annotators](images/dataexplorer/annotators.png?raw=true, "dataexplorer/annotators")
+
+The screen shows a list of available annotators that can be used. Clicking the title of the annotator will result in a popup with additional information such as a general description and a listing of the attributes that will be added by this annotator.
+Using the checkboxes multiple annotaotrs can be selected for one run, which is starten by clicking the "annotate" button. If preferred a copy of the dataset can be created with the annotations added to this copy, leaving the original entity as it is.
+Annotated fields will be added to the entity in a compound attribute.
+
+On the right hand side of the screen a list of unavailable annotators is shows, the reason why they are unavailable is shown in the list, this can for example be due to a resource being unavailble or an atrribute needed to map the entity and resource to each other missing.
+The gear icon trailing every annotator in the list can be used to configure the settings for this annotator.
+
+##### Aggregation mod
+
+The aggregation mod allows you to produce counts for queries on the data.
```

Comment:
'allows you to aggregate' or 'produce aggregates' or 'produce aggregated counts', but the word aggregate should return somewhere I think